src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -611,18 +612,18 @@ case class PLabelDef(name: String) extends PDefLikeLabel
 case class PLabelUse(name: String) extends PUseLikeLabel
 
 
-sealed trait PPackegeNode extends PNode {
+sealed trait PPackageNode extends PNode {
   def name: PPkg
 }
 
-trait PDefLikePkg extends PPackegeNode
-trait PUseLikePkg extends PPackegeNode
+trait PDefLikePkg extends PPackageNode
+trait PUseLikePkg extends PPackageNode
 
 case class PPkgDef(name: PPkg) extends PDefLikePkg
 case class PPkgUse(name: PPkg) extends PUseLikePkg
 
 
-case class PWildcard() extends PDefLikeId with PUseLikeId {
+case class PWildcard() extends PDefLikeId with PUseLikeId with PUnkLikeId {
```

Comment:
What is the difference between `PBlankIdentifier` and `PWildcard`?

Reply:
`PBlankIdentifier` and `PWildcard` in essence represent the same concept but they fall into different syntactic categories. `PWildcard` is an identifier (i.e. a subtype of `PIdnNode`) whereas `PBlankIdentifier` is a subtype of `PAssignee` and thus, it is also a `PExpression`. The reason for this seemingly artificial divide between the two is that declaration statements (i.e. `PVarDecl`, `PShortVarDecl` and `PConstDecl`) expect the LHS of the operation to be a vector of ids:
```scala
case class PShortVarDecl(right: Vector[PExpression], left: Vector[PUnkLikeId], addressable: Vector[Boolean]) extends PSimpleStmt with PGhostifiableStatement
```
while assignment expressions expect a vector of expressions (more precisely, a vector of `PAssignee`s):
```scala
case class PAssignment(right: Vector[PExpression], left: Vector[PAssignee]) extends PSimpleStmt with PGhostifiableStatement
```
**Obs**: `PWildcard` is a subtype of `PUnkLideId` and the second is a subtype of `PIdnNode`

Reply:
Ah makes sense! Thanks